h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -62,8 +63,14 @@
   private final boolean enableStagedProbabilities; // if set true, staged probabilities from tree agos are returned
   private final boolean enableContributions; // if set to true, will return prediction contributions (SHAP values) - for GBM & XGBoost
   private final int glrmIterNumber; // allow user to set GLRM mojo iteration number in constructing x.
-
   private final PredictContributions predictContributions;
+  private int predictColumnNum; // Count number of columns that are found in input data row and gam predictor columns
+
+  public void setPredictColumnNum(int predNum) throws IOException {
```

Comment:
this exposes a public API for the sake of one algo, no user will ever call this, either we need to make this private or at least package private
also I think this functionality should be hidden inside GamMojoModel